documentation/dev/plugins.rst

```diff
@@ -174,9 +174,30 @@ Config settings can be registered by setting the (optional) ``__settings__`` att
             "description": "Color used to override the default plugin color.",
             "level": "info",
         },
-    ]
+    }
 
-You might want to override some FlexMeasures configuration settings from within your plugin. Some examples for possible settings are named on this page, e.g. the custom style (see above) or custom logo (see below). There is a `record_once` function on Blueprints which can help with this. Example:
+Alternatively, use ``from my_plugin import __settings__`` in your plugin module, and create ``__settings__.py`` with:
```

Comment:
Technically, you can have your settings in a module with any name you want and import them with:

```
from my_plugin import whatever_settings as __settings__
```

But a Python programmer should understand that.

Reply:
True. I also tried having FlexMeasures look specifically for a `__settings__.py` in the plugin module, but I couldn't figure out how to do that. That would have saved the `import` statement, while requiring an explicit filename.